go.mod

```diff
@@ -4,18 +4,61 @@ go 1.12
 
 require (
 	github.com/ardielle/ardielle-go v1.5.2
-	github.com/gogo/protobuf v1.2.2-0.20190730201129-28a6bbf47e48
-	github.com/json-iterator/go v1.1.7 // indirect
+	github.com/davecgh/go-spew v1.1.1
+	github.com/gogo/protobuf v1.3.1
 	github.com/sirupsen/logrus v1.4.2
-	github.com/stretchr/testify v1.3.0
-	github.com/yahoo/athenz v1.8.23
-	github.com/yahoo/k8s-athenz-syncer v0.1.1
+	github.com/stretchr/testify v1.4.0
+	github.com/yahoo/athenz v1.8.40
+	github.com/yahoo/k8s-athenz-syncer v0.1.6
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
-	istio.io/api v0.0.0-20190906131201-ca4ba7013e1c
-	istio.io/istio v0.0.0-20190909170145-ef9542f41bf4
-	k8s.io/api v0.0.0-20190817221950-ebce17126a01
-	k8s.io/apimachinery v0.0.0-20190817221809-bf4de9df677c
-	k8s.io/client-go v10.0.0+incompatible
-	k8s.io/klog v0.4.0 // indirect
-	k8s.io/kube-openapi v0.0.0-20190816220812-743ec37842bf // indirect
+	istio.io/api v0.0.0-20200107183329-ed4b507c54e1
+	istio.io/istio v0.0.0-20200220235129-f288658b710d
+	istio.io/pkg v0.0.0-20191030005435-10d06b6b315e
+	k8s.io/api v0.18.0
+	k8s.io/apimachinery v0.18.0
+	k8s.io/client-go v15.0.0+incompatible
 )
+
+// Kubernetes makes it challenging to depend on their libraries. To get around this, we need to force
+// the sha to use. All of these are pinned to the tag "kubernetes-1.16"
+replace k8s.io/api => k8s.io/api v0.0.0-20191003000013-35e20aa79eb8
```

Comment:
I made an update to remove all the replaces as many as possible.

Reply:
Left replace is there because istio 1.5.8 library pulls k8s dependencies higher than 1.18 (go client, api, etc..), pinned to version 17.14 which cluster will be upgraded to.